docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,7 +5,16 @@
    "id": "06bdcdfc",
```

Comment:
<p>or due to the dataset's nature it has identical-looking samples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>